velox/docs/configs.rst

```diff
@@ -681,12 +681,47 @@ These semantics are similar to the `Apache Hadoop-Aws module <https://hadoop.apa
      - Type
      - Default Value
      - Description
+   * - fs.azure.account.auth.type.<storage-account>.dfs.core.windows.net
+     - string
+     - SharedKey
+     - Specifies the authentication mechanism to use for Azure storage accounts. 
+       **Allowed values:**: 
+        "SharedKey": Uses the storage account name and key for authentication.
+        "OAuth": Utilizes OAuth tokens for secure authentication.
+        "SAS": Employs Shared Access Signatures for granular access control.
+     -
```

Comment:
@zhli1142015 doc build is failing. Likely due to this extra `-`. You can verify by running `make html` inside `velox/velox/docs`

Reply:
Removing the extra `-` for each config after the description works.